src/frontend/js/utils/cunningham-tokens.ts

```diff
@@ -0,0 +1,117 @@
+export const tokens = {
```

Comment:
I'm wondering if the TS file is needed? 🤔

Reply:
I thought we would need our colors in typescript.
I can remote it for now and we would add it when we need it.

Reply:
+1 we will generate it when we will need it